README.md

```diff
@@ -1,2 +1,181 @@
 # reference-architectures
-Get up and running quickly with one of our reference architecture using our cold start process.
+
+Get up and running quickly with one of our reference architectures using our fully automated cold-start process.
+
+__NOTE:__ This project is under active development and subject to change. Please [file issues](https://github.com/cloudposse/reference-architectures/issues/new) for all bugs encountered.
+
+## Table of Contents
+
+- [reference-architectures](#reference-architectures)
+  - [Table of Contents](#table-of-contents)
+  - [Known Limitations](#known-limitations)
+  - [High Level Overview](#high-level-overview)
+  - [Architecture](#architecture)
+    - [Assumptions](#assumptions)
+    - [Checklist](#checklist)
+    - [1. Provision Root Account](#1-provision-root-account)
+    - [2. Provision Subaccounts](#2-provision-subaccounts)
+    - [3. Delegate DNS](#3-delegate-dns)
+  - [Next Steps](#next-steps)
+  - [Getting Help](#getting-help)
+
+## Known Limitations
+
+- AWS does not support programmatic deletion of accounts. This means that if you use this project to create the account structure, terraform is not able to completely destroy it.
+- AWS by default only permits one subaccount. This limit can be easily increased for your organization, but can take up to several days.
+
+## High Level Overview
+
+You can provision the basic reference architecture in 3 "easy" steps. =)
+
+All accounts will leverage our [`terraform-root-modules`](https://github.com/cloudposse/terraform-root-modules/) to get started.
+
+This process involves using terraform to generate the code (`Dockerfile`, `Makefile`, `terraform.tfvar`, etc) you will use to manage your infrastructure. 
+
+This is a "bootstrap" process. You do it once and then you throw *this* repo away.
```

Comment:
Maybe we need to elaborate why `you throw *this* repo away` (literally or figuratively)